Documentation/getting-started-test-app.md

```diff
@@ -0,0 +1,104 @@
+
+Developers using ad ID should get the value from the API each time it is used, as permissions for ad tracking and/or the value of the ID itself may be changed at any time.
+
+To detect and handle changes in ad ID value or tracking authorization state, it may be helpful to use:
+- A getter helper method that all requests for ad ID value are routed through.
+- A check in the app lifecycle foreground event (or equivalent).
+
+In some Android environments, the ad ID tracking authorization is controlled using a toggle where the existing ad ID value remains unchanged:
+
+[<img src="./assets/old_adid_setting_optin.png" alt="Old ad ID settings page - opt-in state" width="210"/>](./assets/old_adid_setting_optin.png)
+[<img src="./assets/old_adid_setting_optout_prompt.png" alt="Old ad ID settings page - opt-out prompt" width="210"/>](./assets/old_adid_setting_optout_prompt.png)
+[<img src="./assets/old_adid_setting_optout.png" alt="Old ad ID settings page - opt-out state" width="210"/>](./assets/old_adid_setting_optout.png)
+
+Based on Android emulator (Pixel_3a_API_32_arm64-v8a) testing for the ad ID toggle view using the Google Mobile Ads Lite SDK:
+- Ad ID settings can be accessed through the device settings: Settings -> Privacy -> Ads. On this page:
+    - Users are opted-in to ad ID tracking by default.
+    - The ad ID value can be reset; replacing the old value with a new one.
+    - Ad tracking authorization status can be changed using the toggle.
+    - The ad ID value can be viewed.
+- There is no ad tracking permission prompt shown to the user on first app launch.
+- Changes in ad tracking authorization status or ad ID value do not terminate the app.
+- When ad tracking is limited, the Mobile Ads Lite SDK still returns the device's current valid ad ID; see Google's API reference on [`getId()`](https://developers.google.com/android/reference/com/google/android/gms/ads/identifier/AdvertisingIdClient.Info#public-string-getid) for the latest guidance on accessing the ad ID value and expected return values.
+    - Therefore, using the [`isLimitAdTrackingEnabled()`](https://developers.google.com/android/reference/com/google/android/gms/ads/identifier/AdvertisingIdClient.Info#isLimitAdTrackingEnabled()) API to determine tracking authorization status before accessing ad ID value is recommended.  
+
+In other Android environments, the ad ID tracking authorization is controlled using a delete, where the existing ad ID value is deleted and replaced by an all-zero value until it is recreated by user selection:
+
+[<img src="./assets/new_adid_setting_optin.png" alt="New ad ID settings page - opt-in state" width="210"/>](./assets/new_adid_setting_optin.png)
+[<img src="./assets/new_adid_setting_optout.png" alt="New ad ID settings page - opt-out state" width="210"/>](./assets/new_adid_setting_optout.png)
+
+## Implementation examples
+Mobile Ads Lite SDK: Using a getter to return the ad ID value. Key points to note:
```

Comment:
Is this example using Mobile Ads SDK or the Lite version? 
If non-lite version, can we have sub-sections like this (note the headers and the header level):
## Android Ads SDKs
### Google Mobile Ads SDK - implementation example
Using a getter to return the ad ID value. Key points to note:
...
### Google Mobile Ads Lite SDK
The [Google Mobile Ads Lite SDK](https://developers.google.com/admob/android/lite-sdk) is a way to use ads APIs without including the full size Google Mobile Ads SDK.
...
### AndroidX Ads SDK
...

Reply:
Should be addressed with update to format, consolidating Google Mobile Ads Lite SDK section description + implementation example